src/tests/asyncTests.ts

```diff
@@ -235,15 +228,17 @@ export class AsyncTests {
         passRate: calculatePercentage(globalTests.passed, testResults.length),
         failRate: calculatePercentage(globalTests.failed, testResults.length),
         skipRate: calculatePercentage(globalTests.skipped, testResults.length),
-        testStartTime: formatStartTime(testRunSummary.StartTime, 'ISO'),
-        testExecutionTimeInMs: testRunSummary.TestTime ?? 0,
-        testTotalTimeInMs: testRunSummary.TestTime ?? 0,
+        testStartTime: testRunSummary?.StartTime
```

Comment:
Under what situations the fields including `StartTime`, `TestTime` and `UserId` do not exist? If there are scenarios that those fields do not exist, why do not we declare the [fields](https://github.com/forcedotcom/salesforcedx-apex/blob/main/src/tests/types.ts#L258) as optional and claim the scenarios when it is optional. It has always been confusing to me.